src/js/components/TextArea/__tests__/TextArea-test.tsx

```diff
@@ -144,7 +144,7 @@ describe('TextArea', () => {
       const textArea = component.getByPlaceholderText('item');
 
       fireEvent.keyDown(textArea, keyEvent);
-      expect(capturedEvent).toEqual(expect.objectContaining(keyEvent));
+      expect(capturedEvent).toEqual(null);
```

Comment:
Can we avoid changing this test, this change is a sign that we may have broken existing functionality?